oneview_redfish_toolkit/tests/blueprints/test_zone.py

```diff
@@ -91,6 +107,32 @@ def test_get_zone(self, g_mock):
         self.assertEqual("application/json", response.mimetype)
         self.assertEqualMockup(zone_mockup, expected_result)
 
+        # Tests oneview's calls
+        spt_uuid = server_profile_template["uri"].split("/")[-1]
+        g_mock.oneview_client.server_profile_templates.get.\
+            assert_called_with(spt_uuid)
+        g_mock.oneview_client.connection.get.\
+            assert_called_with("/rest/index/trees{}?childDepth=1".format(
+                logical_enclosure["enclosureGroupUri"]))
+        g_mock.oneview_client.index_resources.get_all.\
+            assert_called_with(category='drives', count=10000)
+        g_mock.oneview_client.logical_enclosures.get.\
+            assert_called_with(logical_enclosure["uri"])
+        g_mock.oneview_client.server_hardware.get_all.assert_called_with(
+            filter=self._get_sh_filter(logical_enclosure))
```

Comment:
I think it is better write the expectation as a literal string instead of create a separeted method with a logic to do that, because it seems like the test will need a test to validate the new logic inside the method, do you understand my thinking?